### PR TITLE
Add RPM upgrade solutions for two common errors

### PR DESCRIPTION
**Pull request by domcleal**
Rewritten and updated the SCL pre-requisites section as the source for
SCLs used in 1.10 has changed.

### CODE REVIEW

**Comment by ekohl** [comment]
:+1: I just ran into the rubygem-thin issue and can confirm it solves it.

**Comment by ekohl** [comment]
Maybe also mention the ruby193-rubygems-1.8.24 issue?

**Comment by domcleal** [comment]
> Maybe also mention the ruby193-rubygems-1.8.24 issue?

Updated to include this.

**Comment by mmoll** [comment]
merged, thanks @domcleal!

File: _includes/manuals/1.10/3.6_upgrade.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ When upgrading across more than one version it is recommended to upgrade to each
 intermediate version and follow all upgrade instructions for the previous versions
 before updating to {{page.version}}.
 
+Please note the following known issue:
+
+* RPM users with `ruby193-rubygems-1.2.4` installed (usually from old Foreman
```
Comment (ekohl) [inline _includes/manuals/1.10/3.6_upgrade.md:11]: Shouldn't this be .24 instead of 4.?

File: _includes/manuals/1.10/3.6_upgrade.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ When upgrading across more than one version it is recommended to upgrade to each
...
+  versions) should run `yum downgrade ruby193-rubygems-1.2.3` to fix potential
```
Comment (ekohl) [inline _includes/manuals/1.10/3.6_upgrade.md:12]: Same with .23

File: _includes/manuals/1.10/3.6_upgrade.md (hunk 1 above)
Comment (mmoll) [inline _includes/manuals/1.10/3.6_upgrade.md:11]: indeed, #500

**Pull request merged**